Wellthier/View Controllers/LoginViewController.h

```diff
@@ -12,6 +12,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface LoginViewController : ViewController
 @property (weak, nonatomic) IBOutlet UITextField *usernameField;
 @property (weak, nonatomic) IBOutlet UITextField *passwordField;
+- (IBAction)gesture:(id)sender;
```

Comment:
nit: would recommend to have an extra line between variable declaration and action declarations.